[zh]sync setup-3 (#33625)
kubernetes/website: content/zh/docs/setup/production-environment/container-runtimes.md

- [containerd](#containerd)
- [CRI-O](#cri-o)
- [Docker Engine](#docker)
- [Mirantis Container Runtime](#mcr)

<!-- 
{{< note >}}
Kubernetes releases before v1.24 included a direct integration with Docker Engine, using a component named _dockershim_. That special direct integration is no longer part of Kubernetes (this removal was [announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation) as part of the v1.20 release). You can read [Check whether Dockershim deprecation affects you](/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/) to understand how this removal might affect you. To learn about migrating from using dockershim, see [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).

If you are running a version of Kubernetes other than v{{< skew currentVersion >}}, check the documentation for that version.
{{< /note >}}

 -->
 {{< note >}}
提示：v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 _dockershim_ 的组件。 
这种特殊的直接整合不再是Kubernetes 的一部分
（这次删除是[宣布](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)作为 v1.20 版本的一部分）。

> 你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/) 以了解此删除可能会如何影响你。 

Reviewer: ```suggestion
你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/)
以了解此删除可能会如何影响你。 
```

要了解如何使用 dockershim 进行迁移，请参阅[从 dockershim 迁移](/docs/tasks/administer-cluster/migrating-from-dockershim/)。

如果你正在运行 v{{< skew currentVersion >}} 以外的 Kubernetes 版本，检查该版本的文档。
{{< note >}}

<!-- body -->
<!-- 
## Install and configure prerequisites

The following steps apply common settings for Kubernetes nodes on Linux. 

You can skip a particular setting if you're certain you don't need it.

For more information, see [Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) or the documentation for your specific container runtime.
 -->

## 安装和配置先决条件

以下步骤将通用设置应用于 Linux 上的 Kubernetes 节点。

如果你确定不需要某个特定设置，则可以跳过它。

> 有关更多信息，请参阅 [网络插件要求](/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) 

Reviewer: ```suggestion
有关更多信息，请参阅[网络插件要求](/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) 
```

If you are running a version of Kubernetes other than v{{< skew currentVersion >}}, check the documentation for that version.
{{< /note >}}

 -->
 {{< note >}}
提示：v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 _dockershim_ 的组件。 
这种特殊的直接整合不再是Kubernetes 的一部分
（这次删除是[宣布](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)作为 v1.20 版本的一部分）。
你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/) 
以了解此删除可能会如何影响你。 

> 要了解如何使用 dockershim 进行迁移，请参阅[从 dockershim 迁移](/docs/tasks/administer-cluster/migrating-from-dockershim/)。

Reviewer 2: nit: This article have Mandarin version.

如果你正在运行 v{{< skew currentVersion >}} 以外的 Kubernetes 版本，检查该版本的文档。

> {{< note >}}

Reviewer: ```suggestion
{{< /note >}}
```

<!-- body -->
<!-- 
## Install and configure prerequisites

The following steps apply common settings for Kubernetes nodes on Linux. 

You can skip a particular setting if you're certain you don't need it.

For more information, see [Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) or the documentation for your specific container runtime.
 -->

## 安装和配置先决条件

以下步骤将通用设置应用于 Linux 上的 Kubernetes 节点。

如果你确定不需要某个特定设置，则可以跳过它。

有关更多信息，请参阅[网络插件要求](/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) 
或特定容器运行时的文档。

<!-- 
### Forwarding IPv4 and letting iptables see bridged traffic

Verify that the `br_netfilter` module is loaded by running `lsmod | grep br_netfilter`. 

To load it explicitly, run `sudo modprobe br_netfilter`.

In order for a Linux node's iptables to correctly view bridged traffic, verify that `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config. For example: 
-->

### 转发 IPv4 并让 iptables 看到桥接流量

通过运行 `lsmod | grep br_netfilter` 来验证 `br_netfilter` 模块是否已加载。

> 若要显式加载此模块，请运行`sudo modprobe br_netfilter`。

Reviewer 2: nit: Missing space before `` `code` ``.

### 将 kubeadm 托管的集群迁移到 `systemd` 驱动

<!-- 
If you wish to migrate to the `systemd` cgroup driver in existing kubeadm managed clusters, follow [configuring a cgroup driver](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/).

## CRI version support {#cri-versions}

Your container runtime must support at least v1alpha2 of the container runtime interface.

Kubernetes {{< skew currentVersion >}}  defaults to using v1 of the CRI API.If a container runtime does not support the v1 API, the kubelet falls back to using the (deprecated) v1alpha2 API instead.
-->

> 如果你希望迁移到现有 kubeadm 托管集群中的 `systemd` cgroup 驱动程序，请按照 [配置 cgroup 驱动程序](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/)。

Reviewer: ```suggestion
如果你希望将现有 kubeadm 托管集群迁移到 `systemd` cgroup 驱动程序，
请按照[配置 cgroup 驱动程序](/zh/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/)操作。
```

## CRI 版本支持 {#cri-versions}

你的容器运行时必须至少支持容器运行时接口的 v1alpha2。

> Kubernetes {{< skew currentVersion >}} 默认使用 v1 的 CRI API。如果容器运行时不支持 v1 API，则 kubelet 回退到使用（已弃用的）v1alpha2 API。

Reviewer: ```suggestion
Kubernetes {{< skew currentVersion >}} 默认使用 v1 的 CRI API。如果容器运行时不支持 v1 API，
则 kubelet 会回退到使用（已弃用的）v1alpha2 API。

```

### 将 kubeadm 托管的集群迁移到 `systemd` 驱动

<!-- 
If you wish to migrate to the `systemd` cgroup driver in existing kubeadm managed clusters, follow [configuring a cgroup driver](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/).

## CRI version support {#cri-versions}

Your container runtime must support at least v1alpha2 of the container runtime interface.

Kubernetes {{< skew currentVersion >}}  defaults to using v1 of the CRI API.If a container runtime does not support the v1 API, the kubelet falls back to using the (deprecated) v1alpha2 API instead.
-->

> 如果你希望将现有 kubeadm 托管集群迁移到 `systemd` cgroup 驱动程序，

Reviewer: ```suggestion
如果你希望将现有的由 kubeadm 管理的集群迁移到 `systemd` cgroup 驱动程序，
```

### containerd

<!--
This section outlines the necessary steps to use containerd as CRI runtime.

Use the following commands to install Containerd on your system:

-->
本节概述了使用 containerd 作为 CRI 运行时的必要步骤。

使用以下命令在系统上安装 Containerd：

<!-- Follow the instructions for [getting started with containerd](https://github.com/containerd/containerd/blob/main/docs getting-started.md). Return to this step once you've created a valid configuration file, `config.toml`.  -->

> 按照[开始使用 containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) 的说明进行操作。 创建有效的配置文件 `config.toml` 后返回此步骤。

Reviewer: ```suggestion
按照[开始使用 containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) 的说明进行操作。
创建有效的配置文件 `config.toml` 后返回此步骤。
```

Use the following commands to install Containerd on your system:

-->
本节概述了使用 containerd 作为 CRI 运行时的必要步骤。

使用以下命令在系统上安装 Containerd：

<!-- 
Follow the instructions for [getting started with containerd](https://github.com/containerd/containerd/blob/main/docs getting-started.md). Return to this step once you've created a valid configuration file, `config.toml`. 
 -->

按照[开始使用 containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) 的说明进行操作。 
创建有效的配置文件 `config.toml` 后返回此步骤。

{{< tabs name="Finding your config.toml file" >}}
{{% tab name="Linux" %}}
<!-- You can find this file under the path `/etc/containerd/config.toml`. -->
你可以在路径 `/etc/containerd/config.toml` 下找到此文件。
{{% /tab %}}
{{< tab name="Windows" >}}
<!-- You can find this file under the path `C:\Program Files\containerd\config.toml`. -->
你可以在路径 `C:\Program Files\containerd\config.toml` 下找到此文件。
{{< /tab >}}
{{< /tabs >}}

<!-- 
On Linux the default CRI socket for containerd is `/run/containerd/containerd.sock`.
On Windows the default CRI endpoint is `npipe://./pipe/containerd-containerd`.

#### Configuring the `systemd` cgroup driver {#containerd-systemd}
--> 

> 在 Linux 上，containerd 的默认 CRI 套接字是`/run/containerd/containerd.sock`。

Reviewer: ```suggestion
在 Linux 上，containerd 的默认 CRI 套接字是 `/run/containerd/containerd.sock`。
```

<!--
This section contains the necessary steps to install CRI-O as a container runtime.

-->
本节包含安装 CRI-O 作为容器运行时的必要步骤。

<!-- To install CRI-O, follow [CRI-O Install Instructions](https://github.com/cri-o/cri-o/blob/main/install.md#readme). -->

> 要安装 CRI-O，请按照 [CRI-O 安装说明](https://github.com/cri-o/cri-o/blob/main/install.md#readme)。

Reviewer: ```suggestion
要安装 CRI-O，请按照 [CRI-O 安装说明](https://github.com/cri-o/cri-o/blob/main/install.md#readme)执行操作。
```

<!-- #### cgroup driver -->

#### cgroup 驱动程序

<!-- CRI-O uses the systemd cgroup driver per default, which is likely to work fine for you. To switch to the `cgroupfs` cgroup driver, either edit `/etc/crio/crio.conf` or place a drop-in configuration in `/etc/crio/crio.conf.d/02-cgroup-manager.conf`, for example: -->

> CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。要切换到 `cgroupfs` cgroup 驱动程序，请编辑 `/etc/crio/crio.conf` 或在 `/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个插入式配置 ，例如：

Reviewer: ```suggestion
CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。要切换到 `cgroupfs` cgroup 驱动程序，
请编辑 `/etc/crio/crio.conf` 或在 `/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个 drop-in 配置 ，例如：
```

```toml
[crio.runtime]
conmon_cgroup = "pod"
cgroup_manager = "cgroupfs"
```

<!-- You should also note the changed `conmon_cgroup`, which has to be set to the value `pod` when using CRI-O with `cgroupfs`. It is generally necessary to keep the cgroup driver configuration of the kubelet (usually done via kubeadm) and CRI-O in sync.
  -->

> 你还应该注意更改的 `conmon_cgroup`，当使用 CRI-O 和 `cgroupfs` 时，必须将其设置为值 `pod`。 通常需要保持 kubelet 的 cgroup 驱动配置（通常通过 kubeadm 完成）和 CRI-O 同步。

Reviewer: ```suggestion
你还应该注意到 `conmon_cgroup` 被更改，当使用 CRI-O 和 `cgroupfs` 时，必须将其设置为值 `pod`。
通常需要保持 kubelet 的 cgroup 驱动配置（通常通过 kubeadm 完成）和 CRI-O 同步。
```

<!-- 
For CRI-O, the CRI socket is `/var/run/crio/crio.sock` by default.

### Docker Engine {#docker}

-->
对于 CRI-O，CRI 套接字默认为 `/var/run/crio/crio.sock`。

> ### Docker 引擎 {#docker}

Reviewer: Docker Engine 可视为专有名词，不要翻译

```suggestion
### Docker Engine {#docker}
```

<!-- {{< note >}}
These instructions assume that you are using the [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) adapter to integrate
Docker Engine with Kubernetes.
{{< /note >}} -->

> 提示：这些说明假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来集成带有 Kubernetes 的 Docker 引擎。

Reviewer: 同上，保留短代码。

> 提示：这些说明假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来集成带有 Kubernetes 的 Docker 引擎。

Reviewer: ```suggestion
以下操作假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来集成将
Docker Engine 与 Kubernetes 集成。
```

<!-- 1. On each of your nodes, install Docker for your Linux distribution as per [Install Docker Engine](https://docs.docker.com/engine/install/#server). -->

> 1. 在你的每个节点上，按照 [安装 Docker 引擎](https://docs.docker.com/engine/install/#server) 为你的 Linux 发行版安装 Docker。

Reviewer: ```suggestion
1. 在你的每个节点上，遵循[安装 Docker 引擎](https://docs.docker.com/engine/install/#server)指南为你的
   Linux 发行版安装 Docker。
```

You can use Mirantis Container Runtime with Kubernetes using the open source [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) component, included with MCR.
-->
[Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR) 是一种商用容器运行时，以前称为 Docker 企业版。
你可以使用 MCR 中包含的开源 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 组件将 Mirantis Container Runtime 与 Kubernetes 一起使用。

<!-- To learn more about how to install Mirantis Container Runtime, visit [MCR Deployment Guide](https://docs.mirantis.com/mcr/20.10/install.html). -->
要了解有关如何安装 Mirantis Container Runtime 的更多信息，请访问 [MCR 部署指南](https://docs.mirantis.com/mcr/20.10/install.html)。
<!-- 
Check the systemd unit named `cri-docker.socket` to find out the path to the CRI socket.
 -->
检查名为 cri-docker.socket 的 systemd 单元以找出 CRI 套接字的路径。

## {{% heading "whatsnext" %}}

As well as a container runtime, your cluster will need a working [network plugin](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model).

> 除了容器运行时，你的集群还需要一个有效的 [网络插件](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model)。

Reviewer: ```suggestion
除了容器运行时，你的集群还需要有效的[网络插件](/zh/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model)。
```

Author: @tengqm updated

```

<!--
If you apply this change， make sure to restart containerd:
-->

> 如果你应用此更改，请确保重新启动 containerd：

Reviewer 2: nit: You've changed the `, ` to `，` in English comment.

<!--
This section contains the necessary steps to install CRI-O as a container runtime.

-->
本节包含安装 CRI-O 作为容器运行时的必要步骤。

<!-- To install CRI-O, follow [CRI-O Install Instructions](https://github.com/cri-o/cri-o/blob/main/install.md#readme). -->

要安装 CRI-O，请按照 [CRI-O 安装说明](https://github.com/cri-o/cri-o/blob/main/install.md#readme)执行操作。

<!-- #### cgroup driver -->

#### cgroup 驱动程序

<!--
 CRI-O uses the systemd cgroup driver per default, which is likely to work fine for you. To switch to the `cgroupfs` cgroup driver, either edit `/etc/crio/crio.conf` or place a drop-in configuration in `/etc/crio/crio.conf.d/02-cgroup-manager.conf`, for example: 
-->

CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。要切换到 `cgroupfs` cgroup 驱动程序，

> 请编辑 `/etc/crio/crio.conf` 或在 `/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个插入式配置 ，例如：

Reviewer: ```suggestion
请编辑 `/etc/crio/crio.conf` 或在 `/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个 drop-in 配置 ，例如：
```

```toml
[crio.runtime]
conmon_cgroup = "pod"
cgroup_manager = "cgroupfs"
```

<!--
 You should also note the changed `conmon_cgroup`, which has to be set to the value `pod` when using CRI-O with `cgroupfs`. It is generally necessary to keep the cgroup driver configuration of the kubelet (usually done via kubeadm) and CRI-O in sync.
-->

你还应该注意到 `conmon_cgroup` 被更改，当使用 CRI-O 和 `cgroupfs` 时，必须将其设置为值 `pod`。
通常需要保持 kubelet 的 cgroup 驱动配置（通常通过 kubeadm 完成）和 CRI-O 同步。

<!-- 
For CRI-O, the CRI socket is `/var/run/crio/crio.sock` by default.

### Docker Engine {#docker}

-->
对于 CRI-O，CRI 套接字默认为 `/var/run/crio/crio.sock`。

### Docker Engine {#docker}

<!-- 
{{< note >}}
These instructions assume that you are using the [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) adapter to integrate
Docker Engine with Kubernetes.
{{< /note >}} 
-->

{{< note >}}

> 以下操作假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来集成将
> Docker Engine 与 Kubernetes 集成。

Reviewer: ```suggestion
以下操作假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来将
Docker Engine 与 Kubernetes 集成。
```

{{< /note >}} 

<!--
 1. On each of your nodes, install Docker for your Linux distribution as per [Install Docker Engine](https://docs.docker.com/engine/install/#server). 
 -->

1. 在你的每个节点上，遵循[安装 Docker 引擎](https://docs.docker.com/engine/install/#server)指南为你的
   Linux 发行版安装 Docker。

<!-- 
2. Install [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd), following the instructions in that source code repository.
-->

> 2. 按照源代码存储库中的说明安装 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd)。

Reviewer: ```suggestion
2. 按照源代码仓库中的说明安装 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd)。
```

 
### Mirantis Container Runtime {#mcr}
 -->
对于 `cri-dockerd`，默认情况下，CRI 套接字是 `/run/cri-dockerd.sock`。
 
### Mirantis 容器运行时 {#mcr}

<!--
[Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR) is a commercially available container runtime that was formerly known as Docker Enterprise Edition.
You can use Mirantis Container Runtime with Kubernetes using the open source [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) component, included with MCR.
-->
[Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR) 是一种商用容器运行时，以前称为 Docker 企业版。
你可以使用 MCR 中包含的开源 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 组件将 Mirantis Container Runtime 与 Kubernetes 一起使用。

<!--
To learn more about how to install Mirantis Container Runtime, visit [MCR Deployment Guide](https://docs.mirantis.com/mcr/20.10/install.html). 
-->
要了解有关如何安装 Mirantis Container Runtime 的更多信息，请访问 [MCR 部署指南](https://docs.mirantis.com/mcr/20.10/install.html)。
<!-- 
Check the systemd unit named `cri-docker.socket` to find out the path to the CRI socket.
 -->
检查名为 cri-docker.socket 的 systemd 单元以找出 CRI 套接字的路径。

## {{% heading "whatsnext" %}}

> As well as a container runtime, your cluster will need a working [network plugin](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model).

Reviewer: 479 注释掉

<!--
This section contains the necessary steps to install CRI-O as a container runtime.

-->
本节包含安装 CRI-O 作为容器运行时的必要步骤。

> <!-- To install CRI-O, follow [CRI-O Install Instructions](https://github.com/cri-o/cri-o/blob/main/install.md#readme). -->

Reviewer: ```suggestion
<!--
To install CRI-O, follow [CRI-O Install Instructions](https://github.com/cri-o/cri-o/blob/main/install.md#readme).
-->
```

要安装 CRI-O，请按照 [CRI-O 安装说明](https://github.com/cri-o/cri-o/blob/main/install.md#readme)执行操作。

<!-- #### cgroup driver -->

#### cgroup 驱动程序

<!--
 CRI-O uses the systemd cgroup driver per default, which is likely to work fine for you. To switch to the `cgroupfs` cgroup driver, either edit `/etc/crio/crio.conf` or place a drop-in configuration in `/etc/crio/crio.conf.d/02-cgroup-manager.conf`, for example: 
-->

CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。要切换到 `cgroupfs` cgroup 驱动程序，

> Suggested change 

Reviewer: ?

-->

你还应该注意到 `conmon_cgroup` 被更改，当使用 CRI-O 和 `cgroupfs` 时，必须将其设置为值 `pod`。
通常需要保持 kubelet 的 cgroup 驱动配置（通常通过 kubeadm 完成）和 CRI-O 同步。

<!-- 
For CRI-O, the CRI socket is `/var/run/crio/crio.sock` by default.

### Docker Engine {#docker}

-->
对于 CRI-O，CRI 套接字默认为 `/var/run/crio/crio.sock`。

### Docker Engine {#docker}

<!-- 
{{< note >}}
These instructions assume that you are using the [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) adapter to integrate
Docker Engine with Kubernetes.
{{< /note >}} 
-->

{{< note >}}
以下操作假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来将
Docker Engine 与 Kubernetes 集成。
{{< /note >}} 

<!--
 1. On each of your nodes, install Docker for your Linux distribution as per [Install Docker Engine](https://docs.docker.com/engine/install/#server). 
 -->

1. 在你的每个节点上，遵循[安装 Docker 引擎](https://docs.docker.com/engine/install/#server)指南为你的
   Linux 发行版安装 Docker。

<!-- 
2. Install [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd), following the instructions in that source code repository.
-->

> 按照源代码仓库中的说明安装 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd)。

Reviewer: ```suggestion
2. 按照源代码仓库中的说明安装 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd)。
```

<!--
For `cri-dockerd`, the CRI socket is `/run/cri-dockerd.sock` by default.
 
### Mirantis Container Runtime {#mcr}
-->
对于 `cri-dockerd`，默认情况下，CRI 套接字是 `/run/cri-dockerd.sock`。
 
### Mirantis 容器运行时 {#mcr}

<!--
[Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR) is a commercially available container runtime that was formerly known as Docker Enterprise Edition.
You can use Mirantis Container Runtime with Kubernetes using the open source [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) component, included with MCR.
-->
[Mirantis Container Runtime](https://docs.mirantis.com/mcr/20.10/overview.html) (MCR) 是一种商用容器运行时，以前称为 Docker 企业版。
你可以使用 MCR 中包含的开源 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 组件将 Mirantis Container Runtime 与 Kubernetes 一起使用。

<!--
To learn more about how to install Mirantis Container Runtime, visit [MCR Deployment Guide](https://docs.mirantis.com/mcr/20.10/install.html). 
-->
要了解有关如何安装 Mirantis Container Runtime 的更多信息，请访问 [MCR 部署指南](https://docs.mirantis.com/mcr/20.10/install.html)。
<!-- 
Check the systemd unit named `cri-docker.socket` to find out the path to the CRI socket.
-->

> 检查名为 cri-docker.socket 的 systemd 单元以找出 CRI 套接字的路径。

Reviewer 2: nit: Missing `` `code` `` syntax.
